tests/Agent/IntegrationTests/UnboundedIntegrationTests/Elasticsearch/ElasticsearchSyncTests.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+using System.Collections.Generic;
+using System.Linq;
+using MultiFunctionApplicationHelpers;
+using NewRelic.Agent.IntegrationTestHelpers;
+using NewRelic.Agent.UnboundedIntegrationTests.ElasticsearchTests;
+using NewRelic.Testing.Assertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace NewRelic.Agent.UnboundedIntegrationTests.Elasticsearch
+{
+    public abstract class ElasticsearchSyncTestsBase<TFixture> : ElasticsearchTestsBase<TFixture>
+        where TFixture : ConsoleDynamicMethodFixture
+    {
+        const string SYNC_MODE = "sync";
+
+        protected ElasticsearchSyncTestsBase(TFixture fixture, ITestOutputHelper output, string clientType) : base(fixture, output, clientType, SYNC_MODE)
+        {
+
+        }
+
+        [Fact]
+        public void IndexAndSearch()
+        {
+            var expectedTransactionName = "OtherTransaction/Custom/MultiFunctionApplicationHelpers.NetStandardLibraries.Elasticsearch.ElasticsearchExerciser/Index";
+
+            var expectedMetrics = new List<Assertions.ExpectedMetric>
+            {
+                new Assertions.ExpectedMetric { metricName = $"Datastore/statement/Elasticsearch/flights/Index", metricScope = expectedTransactionName, callCount = 1 },
+            };
+
+            var metrics = _fixture.AgentLog.GetMetrics().ToList();
+
+            var spanEvents = _fixture.AgentLog.GetSpanEvents();
+
+            var traceId = spanEvents.Where(@event => @event.IntrinsicAttributes["name"].ToString().Equals(expectedTransactionName)).FirstOrDefault().IntrinsicAttributes["traceId"];
+
+            var operationDatastoreSpans = spanEvents.Where(@event => @event.IntrinsicAttributes["traceId"].ToString().Equals(traceId) && @event.IntrinsicAttributes["name"].ToString().Contains("Datastore/statement/Elasticsearch"));
+
+
+            NrAssert.Multiple
+            (
+                () => Assertions.MetricsExist(expectedMetrics, metrics),
+                () => Assert.Equal(1, operationDatastoreSpans.Count())
+            );
+        }
+
+    }
+
+    #region NEST
```

Comment:
This is currently too many tests (3 libraries x 6 .NET frameworks x 2 (sync/async). We can pare down once we decide on how many library versions we want to check.